src/__tests__/EventHolder-test.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Facebook, Inc.
+ * Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
Let's not lose these bits (in all the files). The header should stay how it was (and then we'll update to BSD)